api/lanes.py

```diff
@@ -374,32 +377,167 @@ def lane_for_other_languages(_db, exclude_languages):
     lane.default_for_language = True
     return lane
 
+class LicensePoolBasedLane(Lane):
+    """A lane based on a particular LicensePool"""
+
+    DISPLAY_NAME = None
+    MAX_CACHE_AGE = 14*24*60*60      # two weeks
+
+    # Inside of groups feeds, we want to return a sample
+    # even if there's only a single result.
+    MINIMUM_SAMPLE_SIZE = 1
+
+    def __init__(self, _db, license_pool, full_name,
+                 display_name=None, sublanes=[], invisible=False):
+        self.license_pool = license_pool
+        display_name = display_name or self.DISPLAY_NAME
+        super(LicensePoolBasedLane, self).__init__(
+            _db, full_name, display_name=display_name,
+            sublanes=sublanes
+        )
+
+    def apply_filters(self, qu, facets=None, pagination=None,
+            work_model=Work, edition_model=Edition):
+        """Incorporates additional filters to be run on a query of all Works
+        in the db or materialized view
+
+        :return: query
+        """
+        raise NotImplementedError()
+
+    def featured_works(self, use_materialized_works=True):
+        """Find a random sample of books for the feed"""
+
+        # Lane.featured_works searches for books along a variety of facets.
+        # Because LicensePoolBasedLanes are created for individual works as
+        # needed (instead of at app start), we need to avoid the relative
+        # slowness of those queries.
+        #
+        # We'll just ignore facets and return whatever we find.
+        if not use_materialized_works:
+            query = self.works()
+        else:
+            query = self.materialized_works()
+        if not query:
+            return []
 
-class RecommendationLane(Lane):
-    """A lane of works recommended for a particular work"""
+        return self.randomized_sample_works(query, use_min_size=True)
+
+
+class RelatedBooksLane(LicensePoolBasedLane):
+    """A lane of Works all related to the Work of a particular LicensePool
+
+    Sublanes currently include a SeriesLane and a RecommendationLane"""
+
+    DISPLAY_NAME = "Related Books"
+
+    def __init__(self, _db, license_pool, full_name, display_name=None,
+                 mock_api=None):
```

Comment:
I guess I didn't notice this in your last branch, but can you call this `novelist_api` instead of `mock_api`? There's potentially going to be more than one API going in to the `RelatedBooksLane`, and it also should work fine if you pass in the real API. For an example look at CirculationAPI in api/circulation.py--its constructor has `overdrive`, `threem`, and `axis` arguments and they can be real APIs or mocks.